### DISCUSSION TITLE
迁移安卓APP成功！！

### DISCUSSION THREAD

**Discussion by 2061360308** [Samueli924/chaoxing#406] [Show and tell]
# 迁移安卓APP成功！！

## 现状

目前应该能够进行普通刷课刷视频，但是还不能使用最新的自动答题功能，具体情况见我之前的 issue https://github.com/Samueli924/chaoxing/issues/375

## 资源

请在下方地址获取最新测试APP
https://github.com/CXLite/CXLite/releases/tag/v0.0.1  最新版本已经能够正常在安卓运行

## 总结
这个想法耽搁了有一个月了吧，之前尝试了打包安卓APP的可能性，虽然成功了但是界面和接口改造这些方面任务还是很重，所以迟迟没有完成，另外就是我在担心做很方便的页面导致APP被泛滥使用容易被告侵权。现在这个版本摈弃了之前构建UI页面的想法，而是在界面上显示一个伪终端来完成交互，这么一来需要改动的地方就非常少，并且命令行终端的界面也能提高一下软件的使用门槛。

时间仓促，没来得及整理代码，有没有写说明文档，仅供大家参考吧，后期有空我会整理一下提交整合代码到`Samueli924/chaoxing`的仓库的。

**Comment by Samueli924** [comment]
@2061360308  已经开放了项目的WIKI页，如果可以的话，可以把这个方法作为项目文档放在wiki页里

**Reply by 2061360308** [reply]
> @2061360308  已经开放了项目的WIKI页，如果可以的话，可以把这个方法作为项目文档放在wiki页里

过几天会写的，这几天忙着考试
